storybook/main.js

```diff
@@ -7,6 +12,24 @@ const stories = [
 
 const customEnvVariables = {};
 
+const modulesDir = path.join( __dirname, '../node_modules' );
+
+// Workaround for Emotion 11
```

Comment:
That's a lot of gymnastics to make it work with Storybook :(

Reply:
Yeah it's unfortunate that this is necessary. Storybook is slow to upgrade 🤷